recipes/homer/meta.yaml

```diff
@@ -11,12 +11,10 @@ package:
 
 source:
   - url: http://homer.ucsd.edu/homer/configureHomer.pl
```

Comment:
Please consider enabling HTTPS for this site.
Also it would be a good idea to start using versioning for this tool too. E.g. `http://homer.ucsd.edu/homer/{{ version }}/configureHomer.pl`

Reply:
Those are both perfectly valid points. I couldn't figure out who is the maintainer of Homer, though.
For the SSL issue, I notice it affects all the lab's websites of UCSD (crazy, I know).